src/components/Graph/Graph.jsx

```diff
@@ -1,7 +1,6 @@
-import React from 'react';
 import Node from './Node';
 import Edge from './Edge';
-import List from '../common/List';
+import List from '../Common/List';
 
```

Comment:
Please discuss the other team members before changing folder name (this should be done between sprints to avoid conflicts)